### ISSUE TITLE
Wrong booleanDisjoint where a line is completely contained within a poly

### ISSUE THREAD

**Issue by ath0mas** [Turfjs/turf#1227]
**booleanDisjoint** returns _true_ when testing with a line completely contained within a poly.
Expecting result to be _false_ instead.

Example (turf v5.1.6):
```
var line1 = turf.lineString([[80, 60], [120, 135]]);
var poly1 = turf.polygon([[[50, 50], [50, 150], [150, 150], [150, 50], [50, 50]]]);

turf.booleanDisjoint(line1, poly1);    // returns true, should return false
turf.booleanDisjoint(poly1, line1);    // same

// while contains works properly
turf.booleanContains(poly1, line1);    // returns true, correct
```

**Comment by stebogit** [comment]
Good catch @ath0mas, this is definitely a bug 🐛 .
Is the output of this same case correct for `@turf/boolean-intersects` (which is the opposite of `@turf/boolean-disjoint`)?

@rowanwins @DenisCarriere if we "officially" [decided to follow](https://github.com/Turfjs/turf/issues/1140)  (this is my understanding) the [ESRI definitions](http://edndoc.esri.com/arcsde/9.0/general_topics/understand_spatial_relations.htm) it would be useful to add also the link to their definitions, so understanding and checking the expected output would be easier/faster.

**Comment by ath0mas** [comment]
@stebogit 
Indeed, ESRI definitions state that `Intersects returns the exact opposite result of disjoint.`
but
`@turf/boolean-intersects` does not exist
`@turf/intersect` takes two polygons (testing with a line and a polygon always returns `null`)
`@turf/line-intersect` returns intersecting points (empty result if line is within, not intersecting)

**Comment by stebogit** [comment]
> `@turf/boolean-intersects` does not exist

Ooops... :sweat_smile:

**Comment by ath0mas** [comment]
@DenisCarriere @rowanwins PR #1230 merged, thanks!
and I see it tagged as "5.2.0 milestone": possible to get it in a 5.x release? (and not forced to update to next major 6.x)

**Comment by DenisCarriere** [comment]
Published this change as a patch `v5.1.6`

I think we will be skipping the next minor release and hopping over to `v6.x` right away (too many things will change).

https://unpkg.com/@turf/boolean-disjoint@5.1.6/
